### DISCUSSION TITLE
from_cbor chokes on CBOR tag

### DISCUSSION THREAD

**Discussion by andrewshaf** [nlohmann/json#4195] [Q&A]
auto j = ordered_json::from_cbor(is, true, true, json::cbor_tag_handler_t::store);

instead of storing tagged values as binary container with subtype, it throws the exception 113 (a string was expected as map key, but not found).

Is there a way to work around?

TIA!

**Comment by nlohmann** [comment]
Can you provide a (hopefully small) example input?

**Comment by andrewshaf** [comment]
D901F4D901F5A100636F7061

CBOR Playground (nemo157.com)<https://cbor.nemo157.com/#type=hex&value=D901F4D901F5A100636F7061%0A>
